Clother/View Models/ClothesViewModel.swift

```diff
@@ -26,59 +26,120 @@ class ClothesViewModel {
     }
     
     // TODO: Creates an outfit item and stores it in our application.
-    func createOutfit(top: ClothingItem, bottom: ClothingItem, shoes: ClothingItem, other: [ClothingItem]) {
-        
+    func createOutfit(name: String, top: ClothingItem, bottom: ClothingItem, shoes: ClothingItem, other: [ClothingItem]) {
```

Comment:
You should also update outfitsOnDisplay. Set it equal to the changed array of userOutfits. Make a call to saveClothing to achieve this. On the last line.

Reply:
I also forgot to add a dedicated saveClothes method to the class, please create one and make it act similarly to saveOutfit.